.github/workflows/healthchecks_surrealdb_ci.yml

```diff
@@ -40,7 +40,7 @@ jobs:
           surrealdb_username: root
           surrealdb_password: root
           surrealdb_auth: true
-          surrealdb_strict: 
+          surrealdb_strict:  
```

Comment:
do we need this space here?

Reply:
If you wait a few minutes, we could simply remove this field in the next release of this GitHub action

Reply:
There is no rush, I just wanted to check whether this was an accidental change or something done on purpose.

Reply:
Done